specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2023-03-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2046 @@
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegionSkuDetail"
+          },
+          "description": "List of Sku Detail",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "LocationInfo": {
+      "type": "object",
+      "description": "Information about location (for example: features that it supports)",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location name"
+        },
+        "capabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          },
+          "description": "List of capabilities",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "SkuDetail": {
+      "type": "object",
+      "title": "Sku Detail",
+      "description": "Information about Sku",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The type of RedisEnterprise cluster to deploy. Possible values: (Enterprise_E10, EnterpriseFlash_F300 etc.)",
+          "enum": [
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500"
+          ],
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Capability": {
+      "type": "object",
+      "title": "Capability",
+      "description": "Information about the features the location supports",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Feature name"
+        },
+        "value": {
+          "type": "boolean",
+          "description": "Indicates whether feature is supported or not"
+        }
+      }
+    },
+    "RegionSkuDetail": {
+      "title": "Region Sku Detail",
+      "description": "Details about the location requested and the available skus in the location",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string",
+          "description": "Resource type which has the SKU, such as Microsoft.Cache/redisEnterprise"
+        },
+        "locationInfo": {
+          "$ref": "#/definitions/LocationInfo",
+          "description": "Details about location and its capabilities"
+        },
+        "skuDetails": {
+          "$ref": "#/definitions/SkuDetail",
+          "description": "Details about available skus"
+        }
+      }
+    },
+    "FlushParameters": {
+      "title": "Flush all the keys from the current and all its linked databases",
+      "type": "object",
+      "properties": {
+        "ids": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The resource IDs of the database resources to be flushed"
```

Comment:
>flushed [](http://example.com/codeflow?start=75&length=7)

The resource identifiers of all the database resources in the georeplication group to be flushed.

Reply:
I assume we're doing this because we want to do a linked access check on them? I'm only half-convinced about whether this reasoning makes sense. It doesn't seem like a particularly easy-to-use API, unfortunately. That probably won't matter much for people doing it through the portal.

Reply:
Yes, we want to perform a linked access check on the databases as we are essentially deleting the users data.
Jordon and Kyle wanted to do this as this is quite a dangerous operation
And yeah it should not matter for the people doing it in portal